plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/bean/CoinmateBeanV2.java

```diff
@@ -50,9 +50,9 @@ public void setDate(Date date) {
 
     @Parsed(field = "Type detail")
     public void setTypeDetail(String typeDetail) {
-        if ("BUY".equals(typeDetail) || "QUICK_BUY".equals(typeDetail)) {
+        if ("BUY".equals(typeDetail) || "QUICK_BUY".equals(typeDetail) || "MARKET_BUY".equals(typeDetail)) {
```

Comment:
please make all these strings  to one common place for all coinmate beans as static final strings.